sphinxcontrib/confluencebuilder/config/defaults.py

```diff
@@ -79,6 +79,10 @@ def apply_defaults(app):
         if conf.confluence_file_suffix.endswith('.'):
             conf.confluence_file_suffix = '.conf'
 
+    if conf.confluence_table_width:
+        if conf.confluence_table_width is None:
+            conf.confluence_table_width = 760
```

Comment:
Please restore the `CONFLUENCE_DEFAULT_V2_TABLE_WIDTH` define in its original location and replace this `720` with the define.

Reply:
Done.